doc/manual/src/glossary.md

```diff
@@ -29,6 +51,17 @@
     store object by downloading a pre-built version of the store object
     from some server.
 
+  - [substituter]{#gloss-substituter}\
+    A *substituter* is a store other than `/nix/store` from which Nix will
+    copy a store path instead of building it.  Nix will not copy a store
+    path from a remote store unless one of the following is true:
```

Comment:
```suggestion
    copy a store path instead of building it.  Nix will only copy a store
    path from a remote store if one of the following is true:
```

Reply:
Apparently I can no longer "commit suggestion" on this, so I applied it manually (and moved the word "only" so it modifies "if" rather than modifying "copy").